locales/en/messages.json

```
@@ -7675,5 +7675,12 @@
     "programmingFailedNotification": {
         "message": "FC programming failed",
         "description": "Notification message when programming is unsuccessful"
+    },
+    "osdTextElementLidar": {
+        "message": "Sonar",
+        "description": "Displays distance from sonar sensor in cm"
+    },
+    "osdDescElementLidar": {
+        "message": "Displays distance from sonar sensor in cm"
```

Reuse existing message

```suggestion
        "message": "Displays distance from sonar sensor in cm"
        "message": "$t(osdTextElementLidar)",
        "description": "Don't translate!!!"
```